scenarios/kubernetes_e2e_test.py

```diff
@@ -531,7 +531,8 @@ def expect_normal_gcs(path):
                         try:
                             kubernetes_e2e.main(args)
                         except RuntimeError as err:
-                            if not err.message.startswith('Failed to get shared build location'):
+                            if not str(err.message)\
```

Comment:
is this actually valid? usually I think I see something like
```python
if not (
  str(err.message).startswith(
    ...))

Reply:
BaseException.message is a property that's defined to be a string. It's deprecated, and `str(err)` is preferrered.